src/planetmintgo.c

```diff
@@ -50,12 +50,16 @@ volatile uint8_t myStack[MY_STACK_LIMIT] = {0};
 
 uint8_t* getStack( size_t size )
 {
+#ifndef LINUX_MACHINE
```

Comment:
Wouldn't it be better if we remove malloc in Linux? I think it would be better if the codes were the same on both sides... Using free can be confusing in that way.

Reply:
true! I just pushed